hw2/04-background.html

```diff
@@ -1,28 +1,28 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>04 - Background</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="04-background.js"></script>
   </head>
-  <body>
-    <main>
-      <h1>Exercise 04 - Background</h1>
+  <body class="d-flex justify-content-center align-content-center flex-column p-5 m-5 mt-0">
+    <main class="m-5 bg-white p-3 mt-2 rounded flex-row mx-auto w-75">
+      <h1 class="mb-4">Exercise 04 - Background</h1>
       <p>
-        The background color will change every few seconds (as indicated below).
-        You can stop this behavior and you can also change how often the
-        background color changes.
+        The background color will change every few seconds (as indicated below). You can stop this
+        behavior and you can also change how often the background color changes.
       </p>
-      <div>
-        <label>Background change interval (seconds): </label>
-        <input />
+      <div class="">
+        <label class="d-block" for="interval-time">Background change interval (seconds): </label>
+        <input class="d-block w-50" type="number" id="interval-time" />
```

Comment:
It wasn't specified in the description, but you could set the min value for the input to 0 or 1.

Reply:
I did see that you handle invalid input in the js, but this way its more resilient and no confusion for the user.